src/components/editable-list/Secrets.vue

```diff
@@ -86,6 +86,11 @@ export default {
 </script>
 
 <style scoped>
+@import "../../../node_modules/text-security/dist/text-security-disc.css";
```

Comment:
You should be able to import ~text-security...

Reply:
No, I guess tilde (~) paths doesn't work with import statements. At least it didn't work here.

Reply:
I'm not sure how the syntax is exactly, but since this project is using vue-cli and because of that webpack it should be supported. I'm pretty sure @lnikell can say something about that.

IMHO it's bad to import it like that.